app/controllers/manifests_controller.rb

```diff
@@ -8,12 +8,10 @@ def create
     @manifest = Manifest.new(content: manifest_params)
 
     if @manifest.save && validate_manifest(manifest_params)
```

Comment:
this introduces a race condition. i.e., the manifest could be saved w/o being validated against the schema.

Look at https://github.com/18F/e-manifest/commit/aa7d449e2d456bf6c4ca0224f5ba17196d0701ec